npc/re/jobs/2e/rebellion.txt

```diff
@@ -0,0 +1,1093 @@
+	warp("job_gun", 19, 15);
+	end;
+//------- ??? ??? --------
+}
+
+job_gun,19,24,3	script	Elwin Conick#1	4_F_REBELLION,2,2,{
+	if (Class != Job_Gunslinger) {
+		mes("[Elwin Conick]");
+		mes("If you don't want to join Rebellions, just get out.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (questprogress(12341) == 1) {
+		mes("[Elwin Conick]");
+		mes("Do you to join us Rebellions?");
+		next;
+		mes("^0000ffThe short woman with keen eyes stares into your eyes.^000000");
+		next;
+		mes("[Elwin Conick]");
+		mes("Sign here.");
+		next;
+		mes("^0000ffThe undersigned party [_____] will not speak of the test even if he or she suffers injury, mutilation, or death in an unforeseen accident that may occur during the test.^000000");
+		next;
+		select("WHAT?!");
+		mes("[Elwin Conick]");
+		mes("Do you want join us or not?");
+		next;
+		if (select("I give up.", "...I'll sign.") == 1) {
+			mes("[Elwin Conick]");
+			mes("Your choice.");
+			close2;
+			warp("job_gun", 81, 29);
+			end;
+		}
+		mes("[Elwin Conick]");
+		mes("Good. Now, follow me.");
+		close2;
+		warp("job_gun", 117, 41);
+		end;
+	}
+	if (questprogress(12342) == 1) {
+		mes("[Elwin Conick]");
+		mes("Do you want to try again?");
+		next;
+		if (select("I give up.", "Do it.") == 1) {
+			mes("[Elwin Conick]");
+			mes("Your choice.");
+			close2;
+			warp("job_gun", 81, 29);
+			end;
+		}
+		mes("[Elwin Conick]");
+		mes("Follow me.");
+		close2;
+		warp("job_gun", 117, 41);
+		end;
+	}
+	if (questprogress(12343) == 1) {
+		mes("[Elwin Conick]");
+		mes("You still have Ivan's shooting test, do you want to try again?");
+		next;
+		if (select("I give up.", "Do it.") == 1) {
+			mes("[Elwin Conick]");
+			mes("It's not worth to give up, now go out and relax.");
+			close2;
+			warp("job_gun", 81, 29);
+			end;
+		}
+		mes("[Elwin Conick]");
+		mes("Remember to give your best.");
+		close2;
+		warp("job_gun", 120, 130);
+		end;
+	}
+	if (questprogress(12343) > 1) {
+		mes("[Elwin Conick]");
+		mes("Ivan's shooting test have not finished yet, do you want to try again?");
+		next;
+		if (select("I give up.", "Do it.") == 1) {
+			mes("[Elwin Conick]");
+			mes("It's not worth to give up, now go out and relax.");
+			close2;
+			warp("job_gun", 81, 29);
+			end;
+		}
+		mes("[Elwin Conick]");
+		mes("Remember to give your best.");
+		close2;
+		warp("job_gun", 120, 130);
+		end;
+	}
+	if (questprogress(12345) == 1) {
+		mes("[Elwin Conick]");
+		mes("Ivan is waiting for you, go find him.");
+		close2;
+		warp("job_gun", 120, 130);
+		end;
+	}
+	mes("[Elwin Conick]");
+	mes("You have no more test to take. Why don't you go out and have a drink?");
+	close2;
+	warp("job_gun", 81, 29);
+	end;
+}
+
+job_gun,117,46,3	script	Elwin Conick#2	4_F_REBELLION,2,2,{
+	if (Class != Job_Gunslinger) {
+		mes("[Elwin Conick]");
+		mes("If you don't want to join Rebellions, just get out.");
+		close2;
+		warp("job_gun", 81, 29);
+		end;
+	}
+	if (getstatus(SC_MONSTER_TRANSFORM, 1) == 2431) {
```

Comment:
```suggestion
	if (getstatus(SC_MONSTER_TRANSFORM, 1) == G_L_SHECIL) {
```

Reply:
done, will be pushed together in next commit